scripts/generate-constraints.py

```diff
@@ -108,9 +108,9 @@ def newArgparser():
                 pinInfo = copy.copy(boardPinInfo[pinName])
             else:
                 print('Missing pin description for', pin, pinName, projectPinInfo, file=sys.stderr)
-                pinInfo['LOC'] = 'fmc.%s' % (pinName)
+                pinInfo['PACKAGE_PIN'] = 'fmc.%s' % (pinName)
                 errorDetected = True
-            pinInfo['name'] = pin
+            pinInfo[u'name'] = pin
```

Comment:
OK. That is because it is read in as JSON. Seems reasonable to keep them consistent.